src/Functions/createLink.js

```diff
@@ -39,8 +39,9 @@ async function generateKey() {
  * @param {Express.Response} res
  */
 export default async function createLink(req, res) {
-  // TODO: Validate input and return a 422 for bad stuff.
-  const { url } = req.body;
+  const { url } = validate(req, v => ({
+    url: v.string().uri().required(),
+  }));
```

Comment:
This validator syntax of a function that takes a return value from a passed function and uses it in the function keeps throwing me for a loop (similar as prior PR), but I dig it even if it makes my head hurt, lol.

Reply:
Yeahhhh, totally! I was looking for something self-contained (rather than an implicit dependency on being passed in a `Joi` object). But it is a little funky!